SpatialGDK/Source/SpatialGDK/Private/Interop/SpatialSender.cpp

```diff
@@ -778,11 +792,11 @@ bool USpatialSender::SendRPC(const FPendingRPCParams& Params)
 			}
 			check(NetDriver->IsServer());
 
-			OutgoingOnCreateEntityRPCs.FindOrAdd(TargetObject).RPCs.Add(Params.Payload);
+			OutgoingOnCreateEntityRPCs.FindOrAdd(TargetObject).RPCs.Add(Payload);
 #if !UE_BUILD_SHIPPING
-			NetDriver->SpatialMetrics->TrackSentRPC(Function, RPCInfo.Type, Params.Payload.PayloadData.Num());
+			NetDriver->SpatialMetrics->TrackSentRPC(Function, RPCInfo.Type, Payload.PayloadData.Num());
 #endif // !UE_BUILD_SHIPPING
-			return true;
+			return ERPCError::Success;
```

Comment:
nit: ERPCError::Success seems odd. Perhaps None? Or should we rename the enum type to ERPCOutcome?

Reply:
ERPCResult

Reply:
Changed to ERPCResult.